docs/photonics/examples/waveguide_modes.py

```diff
@@ -85,14 +85,24 @@
     print(f"Effective refractive index: {mode.n_eff:.4f}")
     mode.show(mode.E.real, colorbar=True, direction="x")
 
-powers_in_waveguide = []
-confinement_factors_waveguide = []
+# -
+
+# The electric or magnetic field intensity can be plotted directly from the mode object
+
+# +
+
+modes[0].plot_intensity(field=modes[0].E)
```

Comment:
maybe just a tiny thing, but we don't need to pass the E field again as a parameter, right? There's just one intensity :thinking:

Reply:
I added a field argument as we also have the magnetic intensity. The electric one is way more common though :smiley: